docs/src/app/components/pages/components/TimePicker/ExampleComplex.jsx

```diff
@@ -3,13 +3,17 @@ import TimePicker from 'material-ui/lib/time-picker/time-picker';
 
 export default class TimePickerExampleComplex extends React.Component {
 
+  constructor(props) {
+    super(props);
+    this.state = {value24: new Date(), value12: new Date()};
```

Comment:
Could we have only one state for this example with a default value null as before.
It would be something like:
```js
this.state = {
  value: null,
};
```